### PR TITLE
minor fixes to cid doc

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md [hunk 1]
```diff
@@ -204,15 +204,18 @@ To find your QuickSight username:
     ------------ | -------------
 
 
-If you see no data in QuickSight Dahsboards, plase check the following:
+If you see no data in QuickSight Dahsboards after 24 hours, plase check the following:
  1) Double check that QuickSight has permissions to read from your CUR bucket.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:208]: This instructions should be moved to the troubleshooting and referenced here. We can move it later

**Review by yprikhodko — APPROVED** [review]

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 1 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:208]: I would say that not having data at the first run is an expected behavior. So probably best to leave it here.

Comment (yprikhodko) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:208]: Not seeing data after 24 hours is not expected behaviour. What described below are exactly troubleshooting steps.

File: content/Cost/200_Labs/200_Cloud_Intelligence/_index.md [hunk 2]
```diff
@@ -36,54 +31,30 @@ Cloud infrastructure provides more agility and responsiveness than traditional I
 
 Cloud resources are disposable, and with a pay-per-use model it requires a strong integration between IT governance and organizational governance. Builders need to be able to operate in a cloud environment that’s agile and safe at the same time.
 
+![Images/CIDframework.png](/Cost/200_Cloud_Intelligence/Images/CIDframework.png?classes=lab_picture_small)
+
+The Cloud Intelligence Dashboards offer various advantages, including, but not restricted to:
+
+ - **User-Friendly**: All the data insights are presented in clear and understandable language, arranged by services, and include high-level summaries.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/_index.md:38]: I'm ok with changing bullet points to text, but please keep bold items the same as they are in rest of our marketing materials: Easy to Use, Secure,  In Depth, Cost Efficient

File: content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md (hunk 1 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/Cost & Usage Report Dashboards/Dashboards/deploy_dashboards.md:208]: Why not. But I would suggest doing that as a separate change.

File: content/Cost/200_Labs/200_Cloud_Intelligence/_index.md (hunk 2 above)
Comment (iakov-aws) [reply content/Cost/200_Labs/200_Cloud_Intelligence/_index.md]: Adjusted.

**Pull request merged**